.travis.yml

```diff
@@ -15,11 +15,23 @@ go:
 
 before_install:
   - |
-      if ! git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(\.md)|(\.png)|(\.pdf)|(\.html)|^(LICENSE)|^(docs)'
+      if （ [ "$TRAVIS_REPO_SLUG" = "kubeedge/kubeedge" ] && ! git diff --name-only $TRAVIS_COMMIT_RANGE ） | grep -qvE '(\.md)|(\.png)|(\.pdf)|(\.html)|^(LICENSE)|^(docs)'
```

Comment:
s/（/(
s/）/)

I cannot understand how this works.

Reply:
haha, bad syntax prevent "then"
sorry , i will fix it.